### PR TITLE
Add folder override for Digital Assets

### PR DESCRIPTION
**Pull request by sdbradley**

### CODE REVIEW

File: app/controllers/spree/admin/digital_assets_controller.rb [hunk 1]
```diff
@@ -41,13 +42,18 @@ def spree_folder
...
-          @current_folder ||= Spree::Folder.find_by(id: params[:folder_id])
+          id = params.dig(:image, :folder_id) || params.dig(:folder_id)
+          @current_folder ||= Spree::Folder.find_by(id: id)
         end
 
         def current_folder_children
           @current_folder_children = current_folder.try(:children) || Spree::Folder.where(parent_id: nil)
         end
 
+        def current_digital_asset
```
Comment (fastjames) [inline app/controllers/spree/admin/digital_assets_controller.rb]: What is the advantage of this method over using the provided `@digital_asset` that you get from `ResourceController`? This seems to be what's breaking the specs.

Comment (sdbradley) [reply app/controllers/spree/admin/digital_assets_controller.rb]: none. I forgot about the `@object` from `ResourcesController`. updated. thanks!

**Comment by aneekam** [comment]
https://app.clubhouse.io/enginecommerce/story/1361/as-a-ecommerce-operator-i-can-upload-images-or-delete-them-from-the-digital-asset-manager

**Comment by fastjames** [comment]
Rebased and force-pushed.

**Comment by fastjames** [comment]
Squash merged.

**Pull request closed without merging**